spec/services/personal_care_needs/importer_spec.rb

```diff
@@ -4,17 +4,15 @@
 
 RSpec.describe PersonalCareNeeds::Importer do
   subject(:importer) do
-    described_class.new(
-      profile: profile,
-      personal_care_needs: personal_care_needs
-    )
+    Moves::Importer.new(moves)
   end
 
-  let(:person) { create :person }
+  let(:person) { create :person, nomis_prison_number: 'G3239GV' }
```

Comment:
(also in previous comment), looks like a good candidate for a factory trait/type